fastlane/Fastfile

```diff
@@ -242,6 +242,11 @@ platform :ios do
     archive
   end
 
+  desc "Create Carthage archive"
+  lane :archive_ios do
+    archive
+  end
```

Comment:
Why the need of this?

Reply:
OOoops, I added that when I was debugging a deployment. Must have gotten carried over from another branch.